R/query.R

```diff
@@ -380,15 +376,10 @@ query_ <- function(data, sql, query = TRUE) {
     if (length(cols_to_return) > 0 && length(cols_to_return) < ncol(out$data)) {
       out <- out %>% verb(select, !!!cols_to_return)
     }
-
   } else {
-
     if (use_quoted_deparsed_expressions) {
-
       out <- out %>% verb(transmute, !!!(quote_full_expressions(final_select_list)))
-
     } else if (!transmute_early) {
-
```

Comment:
Could you [configure lintr](https://github.com/jimhester/lintr#project-configuration) to preserve empty lines? The loss of vertical whitespace makes some sections of this code harder for me to read